### DISCUSSION TITLE
Unsure of doc vs implementation

### DISCUSSION THREAD

**Discussion by sambklein** [scikit-hep/pyhf#2456] [Q&A] [answered]
In the [Modifiers and Constraints](https://pyhf.readthedocs.io/en/v0.7.5/intro.html#tab-modifiers-and-constraints) table the MC stat uncertainty is shown to be parameterised by $\delta^2_b$ in the units of the rate in the bin. So I thought I should pass as input data $\delta^2_b / \nu_{b}$, but this didn't give sensible results or match a simple example I built from scratch. Am I doing something wrong in my cross checks? 

Everything works fine if I pass just the unscaled standard deviation $\delta_b$ instead of the scaled variance. I think the tagged line below actually converts to the scaled variance?

https://github.com/scikit-hep/pyhf/blob/adddb0797c564a0158a8e2e69a58ee1f98604bf7/src/pyhf/modifiers/staterror.py#L95

If I'm correct (dubious) I think the constraint term should be $\prod_b\mathrm{Gaus}\left(a_{\gamma_b} = 1\middle|\,\gamma_b,\delta_b / \nu_b\right)$? And the input should be $\delta_b = \sqrt{\sum_s\delta^2_{sb}}$?

**Comment by alexander-held** [comment, marked as answer]
I think you're correct and the width of the Gaussian in the documentation is wrong. For the question of what you actually should put into the modifier in your model: it should be the absolute statistical uncertainty for a given sample, so no scaling needed. See also #1018.

**Reply by sambklein** [reply]
Ok great, thanks! I'll also make a comment on the issue
